src/sb_pointcloud_processing/launch/line_extractor.launch

```diff
@@ -5,7 +5,8 @@
         <!-- regularization constant for line of best fit -->
         <param name="lambda" value="0" type="double" />
         <!-- density parameters for DBSCAN -->
-        <param name="min_neighbours" value="1" type="int" />
-        <param name="radius" value="80" type="double" />
+        <param name="min_neighbours" value="10" type="int" />
```

Comment:
There's a merge conflict here; we should use the changes from #295 (already merged into master), which configures `frame_id` and other variables from launch file.

Reply:
👌